index.html

```diff
@@ -25,5 +25,99 @@ <h3 class="text2">I'm a Software Developer</h3>
+    <div class="tags"><img src="icons/Tags.png"></div>
+    <div class="see"><img src="icons/see-project.png"></div>
+    <div class="grid-container">
+        <div class="first">
+          <h4>Professional Art Printing Data</h4>
+          <p class="ntext">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+          <nav class="button">
+           <p class="html"><img src="icons/htmll.png"></p>
+           <p class="boot"><img src="icons/bootstrap.png"></p>
+           <p class="ruby"><img src="icons/ruby.png"></p>
+           </nav>
+           <div class="see2"><img src="icons/see2.png"></div>
+        </div>
+        <div class="second">
+          <h4>Professional Art Printing Data</h4>
+          <p class="ntext">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+          <nav>
+           <p class="html"><img src="icons/htmll.png"></p>
+           <p class="boot"><img src="icons/bootstrap.png"></p>
+           <p class="ruby"><img src="icons/ruby.png"></p>
+        </nav>
+           <div class="see2"><img src="icons/see2.png"></div>
+        </div>
+        <div class="third">
+          <h4>Professional Art Printing Data</h4>
+          <p class="ntext">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+          <nav class="button">
+           <p class="html"><img src="icons/htmll.png"></p>
+           <p class="boot"><img src="icons/bootstrap.png"></p>
+           <p class="ruby"><img src="icons/ruby.png"></p>
+           </nav>
+           <div class="see2"><img src="icons/see2.png"></div>
+        </div>
+        <div class="fourth">
+          <h4>Professional Art Printing Data</h4>
+          <p class="ntext">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+          <nav class="button">
+           <p class="html"><img src="icons/htmll.png"></p>
+           <p class="boot"><img src="icons/bootstrap.png"></p>
+           <p class="ruby"><img src="icons/ruby.png"></p>
+           </nav>
+           <div class="see2"><img src="icons/see2.png"></div>
+        </div>
+        <div class="fifth">
+          <h4>Professional Art Printing Data</h4>
+          <p class="ntext">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+          <nav class="button">
+           <p class="html"><img src="icons/htmll.png"></p>
+           <p class="boot"><img src="icons/bootstrap.png"></p>
+           <p class="ruby"><img src="icons/ruby.png"></p>
+           </nav>
+           <div class="see2"><img src="icons/see2.png"></div>
```

Comment:
This is suppose to be a button or a link `a` tab and not an image. Kindly use the button or link tag to create it. 
You also need to implement all the button styling interactions (for all your buttons) like the Active, Hover and Pressed as shown in the design below :point_down: 

![Screenshot from 2023-04-29 07-39-21](https://user-images.githubusercontent.com/65233860/235291245-69d5f17f-1e06-483b-8744-9e69425b4258.png)

Reply:
Hello thank you very much for your review. So should I use button inside the ul and il tag? @jerryowusu